jenkins_build.sh

```diff
@@ -0,0 +1,47 @@
+set -e
+
+REPO_NAME=zenoss-prodbin
+REPO_PATH=$WORKSPACE/${ZENDEV_ENV}/src/github.com/zenoss/${REPO_NAME}
+BRANCH=develop
+# The name of the zendev environment that will be created. This name will be used as the devimg tag as well.
+ZENDEV_ENV=${REPO_NAME}-${JOB_BASE_NAME}
+ZENDEV_REPO=git@github.com:zenoss/zendev.git
+ZENDEV_DIR=zendev
+ZENDEV_BRANCH=zendev2
+ZENDEV_VER=0.2.0
+VENV_DIR=venv
+GO_VER=1.7.4
+
+echo Creating a virtual env...
+virtualenv ${VENV_DIR}
+source ${VENV_DIR}/bin/activate
+git clone ${ZENDEV_REPO} ${ZENDEV_DIR}
+cd ${ZENDEV_DIR};git checkout ${ZENDEV_BRANCH}
+pip install -e .
+zendev version | grep ${ZENDEV_VER} # Check zendev version
```

Comment:
Consider how this might look to someone reviewing the log file who doesn't know what to expect. 
When it works, the script will output "0.2.0" without explanation.  If the actual version of zendev changes, the script will output nothing.  I suggest something like:
```
ZENDEV_VERSION=`zendev version`
echo "zendev version = ${ZENDEV_VERSION}"
```